scripts/run_lighthouse_tests.py

```diff
@@ -17,27 +17,36 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import argparse
+import atexit
+import fileinput
 import os
-import shutil
+import re
 import subprocess
 
 import python_utils
+from scripts import build
 from scripts import common
-from . import install_third_party_libs
+from scripts import install_third_party_libs
+
+_PARSER = argparse.ArgumentParser()
+
+_PARSER.add_argument(
+    '--enable_compression',
+    help='optional; if specified, uses nginx to serve compressed assets',
+    action='store_true')
```

Comment:
Hmm, I would probably use the command with compression as a default and add a possibility to disable it.